tests/jest/unit/modules/ai/assets/js/editor/components/prompt-library-link.test.js

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react';
+import PromptLibraryLink from '../../../../../../../../../modules/ai/assets/js/editor/components/prompt-library-link';
+
+describe( 'PromptLibraryLink', () => {
+	it( 'Should render a link to the library from props', async () => {
+		const { getByRole } = render(
+			<div data-testid="test_id">
```

Comment:
Is the `testid` needed?

Reply:
no, removed the wrapping div all together